WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentListFragment.kt

```diff
@@ -37,18 +39,35 @@ class UnifiedCommentListFragment : Fragment(R.layout.comment_list_fragment) {
     private fun CommentListFragmentBinding.setupContentViews() {
         val layoutManager = LinearLayoutManager(context)
         commentsRecyclerView.layoutManager = layoutManager
-        commentsRecyclerView.addItemDecoration(CommentListItemDecoration(commentsRecyclerView.context))
-        commentsRecyclerView.adapter = UnifiedCommentListAdapter(requireContext())
+        commentsRecyclerView.addItemDecoration(UnifiedCommentListItemDecoration(commentsRecyclerView.context))
+
+//        adapter = UnifiedCommentListAdapter(requireContext())
+//        commentsRecyclerView.adapter = adapter
     }
 
     private fun CommentListFragmentBinding.setupObservers() {
         lifecycleScope.launchWhenStarted {
-            viewModel.commentListItems.collectLatest { pagingData ->
-                commentsRecyclerView.adapter?.let { it -> (it as UnifiedCommentListAdapter).submitData(pagingData) }
-            }
+//            viewModel.uiState.collect { uiState ->
+//                setupCommentsList(uiState.commentsListUiModel)
+//            }
         }
     }
 
+//    private fun CommentListFragmentBinding.setupCommentsList(commentsListUiModel: CommentsListUiModel) {
+//        when (commentsListUiModel) {
+//            is CommentsListUiModel.Data -> {
+//                adapter.submitData(lifecycle, commentsListUiModel.pagingData)
+//            }
+//            else -> {
+//            }
+//        }
+//    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        binding = null
+    }
```

Comment:
I don't see this part being _added_ in the diff of the cherry-picked commit – https://github.com/wordpress-mobile/WordPress-Android/pull/14949/files#diff-d3f669617284dbfa52ea7b4aeb770607c0f206258a309d4140a8a417153a1b02L65, but I see it was already there back then tho.
Just double-checking this re-addition is thus intentional given this was not part of the release/17.6 branch when we did 17.6 back then.